### ISSUE TITLE
Invalid encoding makes background images fail to load in Firefox

### ISSUE THREAD

**Issue by dkeeghan** [filamentgroup/directory-encoder#35]
Using version 0.9.1 on OSX Sierra. 

Firefox and IE11 reports images as not being able to be loaded, hence all background image svgs don't appear (however the `data-grunticon-embed` images do work)

*Output SVG (0.9.1) - BROKEN*:
`data:image/svg+xml;charset&#x3D;US-ASCII,%3Csvg%20data-name%3D%22Layer%201%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%3E%3Ctitle%20fill%3D%22%23343740%22%3Eaccount.colors-white%3C%2Ftitle%3E%3Cpath%20d%3D%22M12%204a4%204%200%200%201-8%200%204.08%204.08%200%200%201%204-4%204%204%200%200%201%204%204zM8%209c-4.7%200-8%202.5-8%206v1h16v-1c0-3.5-3.3-6-8-6z%22%20fill%3D%22%23343740%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E`

*Output SVG (0.8.1) - WORKING*:
`data:image/svg+xml;charset=US-ASCII,%3Csvg%20data-name%3D%22Layer%201%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%3E%3Ctitle%20fill%3D%22%23343740%22%3Eaccount.colors-white%3C%2Ftitle%3E%3Cpath%20d%3D%22M12%204a4%204%200%200%201-8%200%204.08%204.08%200%200%201%204-4%204%204%200%200%201%204%204zM8%209c-4.7%200-8%202.5-8%206v1h16v-1c0-3.5-3.3-6-8-6z%22%20fill%3D%22%23343740%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E`

From what I can see the difference is `charset&#x3D;US-ASCII` vs `charset=US-ASCII`. From what I can see, I don't believe that this is an issue on previous versions of OSX with 0.9.1

**Comment by ttaiyo** [comment]
Echoing the above, SVGs are encoded incorrectly and aren't displayed on FF and IE.

I've manually made the following changes for the time being:

```
/node_modules/directory-encoder/lib/svg-uri-encoder.js

// SvgURIEncoder.prefix = "data:image/svg+xml;charset=US-ASCII,"; // broken
SvgURIEncoder.prefix = "data:image/svg+xml;charset%3DUS-ASCII,";  // fixed
```

**Comment by zachleat** [comment]
Looks like this landed in #39—closing.
